2-patterns_integrations/src/main/java/com/epam/dhontar/aqamp/utils/integrations/slack/entity/SlackMessage.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.dhontar.aqamp.utils.integrations.slack.entity;
+
+public class SlackMessage {
+    private String text;
+
+    public SlackMessage(String text) {
+        this.text = text;
+    }
+
+    public SlackMessage() {
+    }
+
+    public void setText(String text) {
+        this.text = text;
+    }
+
+    public String getText() {
+        return text;
+    }
+}
```

Comment:
Clean this class please. Remove unused methods

Reply:
in progress

Reply:
done